3rdparty/CMakeLists.txt

```diff
@@ -512,6 +523,11 @@ ExternalProject_Add(
   URL               ${RAPIDJSON_URL}
   URL_HASH          ${RAPIDJSON_HASH})
 
+if (ENABLE_INSTALL_MODULE_DEPENDENCIES)
+  install(
+    DIRECTORY ${RAPIDJSON_CMAKE_ROOT}/src/rapidjson-${RAPIDJSON_VERSION}/include
+    DESTINATION .)
```

Comment:
Why are there several different install directories for headers: the root installation directory is used here, others use `MESOS_INSTALL_HEADERS`, while glog uses `${MESOS_INSTALL_LIBRARIES}/mesos/3rdparty`. If this inconsistency is really necessary, it would be nice to include some comments explaining why.

Reply:
Here is a list of *autotools* build directories for `include`:
```
csi elfio mesos process rapidjson stout picojson.h
```
and `lib/mesos/3rdparty/include`:
```
elfio glog google nvidia rapidjson zookeeper concurrentqueue.h picojson.h
```
As you can observe there are duplicates and inconsistency.

Reply:
earlier agreed with @greggomann  about option 1. [here](https://github.com/apache/mesos/pull/363/commits/d90680c00e3332de088484aa162af8b90bebee0a) is commit that fixes this inconsitency.